lua/nvim_lsp/nimls.lua

```diff
@@ -0,0 +1,43 @@
+local configs = require 'nvim_lsp/configs'
+local util = require 'nvim_lsp/util'
+
+configs.nimls = {
+  default_config = {
+    cmd = {"nimlsp",};
+    filetypes = {'nim'};
+    log_level = vim.lsp.protocol.MessageType.Warning;
+    settings = {};
```

Comment:
Now, we don't need this line, if setting is empty table.

Reply:
Resolved in 9a5d9b3.